src/realm/object-store/object_store.cpp

```diff
@@ -234,27 +234,6 @@ uint64_t ObjectStore::get_schema_version(Group const& group)
     return table->get_object(0).get<int64_t>(c_versionColumnName);
 }
 
-StringData ObjectStore::get_primary_key_for_object(Group const& group, StringData object_type)
```

Comment:
These two functions were for working with the pk table that we used pre-core 6 and don't make much sense any more.